keyboards/montex/keymaps/default/config.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2021 NachoxMacho   
```

Comment:
Please remove all the trailing whitespace.

Reply:
Done in all licenses